src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4579,6 +4579,71 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/datasets/{id}/snapshotBuilder/concepts/{domainId}/{searchText}:
+    get:
+      tags:
+        - datasets
+        - repository
+      description: >
+        Searches concepts of the specified domain that match the given searchText.
+      operationId: searchConcepts
+      parameters:
+        - in: path
+          description: The UUID of the dataset.
+          name: id
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - in: path
+          description: A domain id.
+          name: domainId
+          required: true
+          schema:
+            type: string
```

Comment:
Could this be an integer?

Reply:
The reason this is a string is because in OMOP tables the domainId is a string like "Observation", "Drug", etc.